### ISSUE TITLE
DoneJS quick start guide notes

### ISSUE THREAD

**Issue by alexisabril** [donejs/donejs#78]
_Issue by **[daffl](https://github.com/daffl)** from Thursday Oct 01, 2015 at 18:51 GMT_
_Originally opened as https://github.com/donejs/donejs/issues/134_

---
- [ ] Introduction explain all used features and link to them
- [ ] Make video with browser window
- [ ] After... to more detail what guide does
- [ ] Link everywhere to the features page
- [ ] Show pre- and after-production network tab, explain number of files in development
- [ ] Divshot show creating an app name `donejs-chat-<username>`

**Comment by alexisabril** [comment]
_Comment by **[justinbmeyer](https://github.com/justinbmeyer)** from Thursday Oct 01, 2015 at 18:54 GMT_

---
- Setup
- Add bootstrap
  - Add bootstrap / update html
  - **Checkout Hot module swapping**
- Setup Routing
  - create pages (generator stuff)
  - setup basic routes that point at each other
  - **Checkout SSR**
- Finish homepage
  - can-tabs + homepage
  - __Custom elements
  - **ease of install**
- Finish Messages
  - **Real-time**
  - **live list**

**Comment by alexisabril** [comment]
_Comment by **[daffl](https://github.com/daffl)** from Saturday Oct 03, 2015 at 00:22 GMT_

---

The guide on donejs.com is updated and verified with the technology changes we discussed. I was able to remove all unnecessary configuration and code (like creating `app.html`, using `steal-platform` to disable routing or checking for `io`).
It is still in the old structure though. The restructured version should be ready for review on Monday.
